test/tools/libtesteth/boostTest.cpp

```diff
@@ -47,11 +48,10 @@ void createRandomTestWrapper()
 	exit(0);
 }
 
-static std::atomic_bool stopTravisOut;
-void travisOut()
+void travisOut(std::atomic_bool *_stopTravisOut)
```

Comment:
When I make this a reference, the compiler errors type checking the constructor of `std::thread` below.

Reply:
are you talking about createRandomTest test issue ? 
if so I dont think that this is the cause. because it was failing only on this test and other tests were just fine.

Reply:
I have no idea, but will keep removing static variables until the issue disappears.

Reply:
@winsvega however, the issue disappeared when I added some debug outputs in the `maing()` function in `boostTest.cpp`.   Also, the issue disappeared when I removed the `travisOut` thread.  https://circleci.com/gh/ethereum/cpp-ethereum/1078